common/laws/00_distribution_of_power.txt

```diff
@@ -519,6 +552,17 @@ law_landed_voting = {
 			limit = { ai_has_enact_weight_modifier_journal_entries = yes }
 			add = 750
 		}
+		if = {
+			limit = { 
+				OR = {
+					has_law = law_type:law_wealth_voting
+					has_law = law_type:law_census_voting
+					has_law = law_type:law_universal_suffrage
+				}
```

Comment:
The journal entry needs to reflect that. Currently both anarchy and single-party laws are not valid for completion of the journal entry.